pallets/offchain-demo/src/lib.rs

```diff
@@ -0,0 +1,220 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+#[cfg(test)]
+mod tests;
+
+use support::{
+	debug,
+	dispatch::DispatchResult, decl_module, decl_storage, decl_event, decl_error,
+	traits::Get,
+	weights::SimpleDispatchInfo,
+};
+
+use core::convert::{TryInto};
+
+use system::{self as system, ensure_signed, ensure_none, offchain};
+// use serde_json as json;
```

Comment:
If this is really unnecessary, just remove it.

If it is there to show where it would go in other situations, that's fine leave it in. But include a comment explaining when and why it would be used.